### ISSUE TITLE
how to get the acc of every epoch while training?

### ISSUE THREAD

**Issue by lx-onism** [qqwweee/keras-yolo3#701]
please! I recently met a bug with yolov3，when I trained a model after data augmetation. the performance of model is far lower than the one without augmention. but the loss is no problem. so I want to know if the acc is rising after each epoch. thanks very much!!!
